fire/__main__.py

```diff
@@ -0,0 +1,17 @@
+"""
+Module for running python fire as a "main" function and thereby allowing using
+fire with other programs.
+"""
+import importlib
+import sys
+
+import fire
+
+def main():
+  module = sys.argv[1]
+  module = importlib.import_module(module)
+  sys.argv = sys.argv[1:2] + sys.argv[2:]
+  fire.Fire(module)
```

Comment:
Rather than manipulating sys.argv, you can pass the name= and command= arguments to fire.Fire.
`name=module, command=sys.argv[2:]`.